curveStateTest

jonathanraes opened the pull request.

jonathanraes committed:
powerup sate for increasing ball size

jonathanraes committed:
refactor and javadocs

mosly added javadocs and fixed some checkstyle, small refactoring

jonathanraes committed:
setup

working setup using spies and stubbing

jonathanraes committed:
fix compiler error

Danilodum commented on src/test/java/com/funkydonkies/controllers/CurveStateTest.java:
```diff
@@ -0,0 +1,78 @@
+	private PhysicsSpace ps;
+	private RigidBodyControl rb;
+	private PlayState playState;
+	private CurveState spy;
```
might need more descriptive name

jonathanraes committed:
some verifications

jonathanraes committed:
merged and fixed test

jonathanraes committed:
Merge branch 'ccMesh' into curveStateTest

Danilodum commented on src/test/java/com/funkydonkies/controllers/CurveStateTest.java:
```diff
@@ -0,0 +1,103 @@
+		spyCurveState.update(0.01f);
+		
+//		verify(spyCurveState).makeRigidBodyControl();
+//		verify(geom).removeControl(rb);
```
uncommented code shouldn't be present, I guess it's still a WIP?

jonathanraes committed:
added bridge, wont build anymore?

jonathanraes commented:
for some reason the cameradetection tests work on travis but not locally (using mvn)

jonathanraes committed:
added some more, still need to figue out how to test some

jonathanraes committed:
use buttuns to select the camera

sdumasy committed:
Added empty classes for obstacles and targets

jonathanraes committed:
added refresh button

jonathanraes committed:
updated pom to compile using java 7

jonathanraes committed:
fixed tool warnings

jonathanraes committed:
merged + working powerup

merged with master to get up to date code and fixed the supersize powerup so that it is enabled after 5 sec. press p to reset the size

jonathanraes committed:
added the refresh icon

sdumasy committed:
SpawnState refactor and faster ballSpawning with higher combo

sdumasy committed:
working difficulty for ballspawnrate and ballspeed

sdumasy committed:
removed checkstyle errors

sdumasy committed:
removed some bugs

jonathanraes committed:
fixed package name of powerups, added speedincrease start

sdumasy committed:
fixed a few errors

jonathanraes committed:
merged with MegaReFactory to get up to date code

needed the diff state etc

jonathanraes committed:
added invertcontrols and tier2

Commit:
refactored most of methods into smaller chunks

updatePoints still needs some better way of doing, this stuff is built-in, I'm reinventing the wheel.

jonathanraes committed:
powerup growing snowball added

jonathanraes committed:
working growingsnowball

jonathanraes committed:
working growingsnowball using a 2d circle

jonathanraes committed:
some fixes

fixed the speed of the snowball and the forced movement on the wave

sdumasy committed:
some refacotar, keep track of the highest combo

sdumasy committed:
added working spear

sdumasy committed:
add waringlines

Commit:
working with factory

Commit:
fixed time not resetting bug

Commit:
fixed warningLine not detaching

sdumasy committed:
fixed spear not detaching

sdumasy committed:
removed redundant classes

sdumasy committed:
fixed collision error

sdumasy committed:
made warningLine transparent again

sdumasy committed:
easy spawning

sdumasy committed:
spearFactory refactor

sdumasy committed:
added a killerwhale obstacle

sdumasy committed:
adder killeWhale factory

sdumasy committed:
added aditional targets

jonathanraes committed:
begin yeti powerup

Commit:
created spikey ball control and factory classes

not yet tested and no spawning behaviour. also no dedicated mesh yet

jonathanraes committed:
start control

sdumasy committed:
fixed balls not being destroyed

Commit:
Merge branch 'master' into selectingCameraInput

Commit:
removed println

sdumasy committed:
whale jumps less high

jonathanraes committed:
snowball is ghostcontrolled and destroys balls on hit

jonathanraes committed:
Merge branch 'selectingCameraInput' into Beta

Commit:
spikey ball works as expected

spawn behaviour and mesh need to be made and needs to be tested

jonathanraes committed:
added poweruptiers

Commit:
checkstyled

jonathanraes committed:
merged FactoryReflections, adding obstacles

olivier-dikken committed:
Merge pull request #55 from Denpeer/selectingCameraInput

Selecting camera input source

jonathanraes committed:
added yeti

sdumasy committed:
half working polarbear added

jonathanraes committed:
added the spikey ball and fixed invincibility for snowballs

sdumasy committed:
add polarBear

jonathanraes committed:
merged polarbear missing the classes >.>

Commit:
Overpowered thunder

Commit:
fixed thunder warning

jonathanraes committed:
added polar bear

jonathanraes committed:
fixes

jonathanraes committed:
added thunder and activating powerup on target grab

jonathanraes committed:
removed printlines

jonathanraes committed:
more printlns and fixed spawning spacials

Commit:
fixed whale movement

Commit:
Merge branch 'Beta' of https://github.com/Denpeer/ContextProject into Beta

jonathanraes committed:
fexed quid collisions

jonathanraes committed:
Merge branch 'Beta' of https://github.com/Denpeer/ContextProject into Beta

jonathanraes committed:
few bugfixes

jonathanraes committed:
moved growingsnowballcontrol

Commit:
Merge pull request #66 from Denpeer/CustomCurveMesh

Loading Scene

Commit:
merge with master

jonathanraes committed:
fix hitting squid and slower obstacle spawn

jonathanraes committed:
Merge branch 'Beta' of https://github.com/Denpeer/ContextProject into Beta

jonathanraes committed:
slower ball spawn

jonathanraes committed:
made curve white and diabled debug

Commit:
spikey ball collisions now cause combo count to reset

Commit:
cleaned test hierarchy

jonathanraes committed:
invert controls test

jonathanraes committed:
Merge branch 'Beta' of https://github.com/Denpeer/ContextProject into Beta

jonathanraes committed:
increasespawnspeed test

Commit:
changed fishcontrol, will change others

Commit:
Merge branch 'Beta' of https://github.com/Denpeer/ContextProject into Beta

jonathanraes committed:
updated combotest

jonathanraes committed:
merge

jonathanraes committed:
updated apptest

jonathanraes committed:
updated combo test, 100% coverage

sdumasy committed:
refactored some controlls

jonathanraes committed:
tested Difficulty state

jonathanraes committed:
fixed appTest

Commit:
refacted killerwhale

Commit:
refactored squid f&c and thunder f&c and warning line c

also completed checkstyle and javadoc on those classes.BTW f=factory & c=control

Commit:
Merge branch 'RefactoringSquidThunderYeti' into facControlRefactor

Commit:
moved speed to control

Commit:
refactored penguin

Commit:
Merge branch 'Beta' into facControlRefactor

Commit:
refacted killercontrol and penguin

Commit:
refactored yeti factory and control and removed unused delete method from several control classes

Commit:
Merge branch 'facControlRefactor' of https://github.com/Denpeer/ContextProject into facControlRefactor

jonathanraes committed:
playstateTest and fixed combo

jonathanraes committed:
fix displaying combo

sdumasy committed:
tested fishfactory

jonathanraes committed:
updated inputstatetest and spawnStateTest

Commit:
refactored almost all controls to use new abstractControls

Commit:
Merge branch 'facControlRefactor' of https://github.com/Denpeer/ContextProject into facControlRefactor

Commit:
started putting more in the superclass

sdumasy committed:
new fishfactorytest

jonathanraes committed:
removed all fail() statements from the test and fixed penguincontroltest

jonathanraes committed:
Merge branch 'Beta' into facControlRefactor

jonathanraes committed:
defined those vars at start of file

jonathanraes committed:
Merge pull request #71 from Denpeer/facControlRefactor

refactored some controls

jonathanraes committed:
merged beta and fixed the fishfactorytext

jonathanraes committed:
Merge pull request #72 from Denpeer/FishFactoryTest

tested fishfactory

sdumasy committed:
tested spikeybalfactory

sdumasy committed:
added a few more tests

jonathanraes committed:
merged Beta, but shit seems to be completely broken (thunder)

Commit:
collision fish-penguin working, rest no

jonathanraes committed:
fixed yeti-control

sdumasy committed:
polar bear fixed

jonathanraes committed:
Merge branch 'Beta' of https://github.com/Denpeer/ContextProject into Beta

sdumasy committed:
fixed killerWhale

sdumasy committed:
Merge branch 'Beta' of https://github.com/Denpeer/ContextProject.git
into Beta

Conflicts:
	src/main/java/com/funkydonkies/gamestates/SpawnState.java

sdumasy committed:
fixed some obstacles, adjusted time warningline

sdumasy committed:
fixed target controllers

sdumasy committed:
fixed a few tests

sdumasy committed:
added killerWhale and krill tests

jonathanraes committed:
added max on snowball radius

jonathanraes committed:
Merge branch 'Beta' of https://github.com/Denpeer/ContextProject into Beta

jonathanraes committed:
fixed spawn location squid/krill
fixed snowballs hitting targets

sdumasy committed:
fishfactory fully tested

sdumasy committed:
Merge branch 'Beta' of https://github.com/Denpeer/ContextProject.git
into Beta

Conflicts:
	src/main/java/com/funkydonkies/controllers/FishControl.java

sdumasy committed:
fishcontrol fix

sdumasy committed:
fixed many tests, and some refactor in factories / controls

sdumasy committed:
penguin factory tests added

Commit:
merge Beta

Commit:
CurveState refactored, regression tested

Commit:
fixed failing tests

Commit:
Merge pull request #67 from Denpeer/refactorCurveState

refactored CurveState
